src/søknad/steg/1-omdeg/OmDeg.tsx

```diff
@@ -22,6 +30,18 @@ const OmDeg: FC<{ intl: IntlShape }> = ({ intl }) => {
 
   const kommerFraOppsummering = location.state?.kommerFraOppsummering;
 
+  const erBegrunnelseBesvart: boolean =
+    (begrunnelseForSøknad?.svarid === EBegrunnelse.samlivsbruddForeldre &&
+      datoForSamlivsbrudd?.verdi !== undefined) ||
+    (begrunnelseForSøknad?.svarid === EBegrunnelse.samlivsbruddAndre &&
+      datoFlyttetFraHverandre?.verdi !== undefined) ||
+    (begrunnelseForSøknad?.svarid === EBegrunnelse.endringISamværsordning &&
+      datoEndretSamvær?.verdi !== undefined) ||
+    begrunnelseForSøknad?.svarid === EBegrunnelse.aleneFraFødsel ||
+    begrunnelseForSøknad?.svarid === EBegrunnelse.dødsfall ||
+    (begrunnelseForSøknad?.svarid === EBegrunnelse.annet &&
+      begrunnelseAnnet?.verdi !== undefined);
+
```

Comment:
Tar imot forslag for finere kode her 😁

Reply:
Muligens formatering kan hjelpe litt
```javascript
let svarid = begrunnelseForSøknad?.svarid;
  const erBegrunnelseBesvart: boolean = svarid ?
      ((svarid === EBegrunnelse.samlivsbruddForeldre && datoForSamlivsbrudd?.verdi !== undefined)
          || (svarid === EBegrunnelse.samlivsbruddAndre && datoFlyttetFraHverandre?.verdi !== undefined)
          || (svarid === EBegrunnelse.endringISamværsordning && datoEndretSamvær?.verdi !== undefined)
          || (svarid === EBegrunnelse.annet && begrunnelseAnnet?.verdi !== undefined)
          || svarid === EBegrunnelse.aleneFraFødsel
          || svarid === EBegrunnelse.dødsfall)
      : false
  ;
```

Reply:
Kan dette være en funksjon som tar inn svarId og kjører switch/case på hver begrunnelse? 

Vil i så fall være litt lettere å få oversikt for min del, tror jeg :D

Reply:
```
const etBraNavn = (svarId: EBegrunnelse, sivilstatus: ISivilstatus) => {
  switch (svarId) {
    case EBegrunnelse.aleneFraFødsel:
      return sivilstatus.datoForSamlivsbrudd?.verdi !== undefined;
  }
...
```

Reply:
Takk for innspill, @charliemidtlyng  og @blommish ✨ ! 
Noe annet jeg lurer på er om dere ville plassert denne kodebiten i en helper-fil, util-fil eller direkte i samme fil? 😁

Reply:
Jeg dro ut en switch/case funksjon i helpers/omdeg.ts!